samples/apps/spring-petclinic/db.annotations

```diff
@@ -0,0 +1,6 @@
+# tag::db-annotations[]
+service.binding/type="postgresql"
+service.binding/host="path={.metadata.name}.{.metadata.namespace}"
+service.binding/port="path={.spec.template.spec.containers[0].ports[0].containerPort}"
+service.binding="path={.spec.template.spec.volumes[0].secret.secretName},objectType=Secret"
+# end::db-annotations[]
```

Comment:
this file is not used anywhere, can we remove it?

Reply:
indeed, I'll remove it.. thx